src/pkg/cli/new.go

```diff
@@ -97,9 +101,11 @@ func InitFromSamples(ctx context.Context, dir string, names []string) error {
 			}
 		}
 	}
+	if !sampleFound {
+		return fmt.Errorf("sample not found")
```

Comment:
make this a global variable `var ErrSampleNotFound = errors.New("sample not found")` and use it here as well as in commands.go